explorer/assets/css/app.css

```diff
@@ -50,13 +50,45 @@
 }
 
 @layer components {
-	.tooltip {
-		display: none;
-	}
+    .tooltip {
+        display: none;
+    }
+
+    .tooltip[data-show] {
+        position: absolute;
+        display: block;
+        z-index: 9999;
+    }
 
-	.tooltip[data-show] {
-		position: absolute;
-		display: block;
-		z-index: 9999;
-	}
+    /* Progress bar based on https://dev.to/mikekennedydev/progress-bar-text-colour-1gg6
+     * Foreground color removed to use the accent colors */
+    .progress-with-labels {
+        width: 100%;
+        position: relative;
+        font-weight: bold;
+
+        .progress-foreground {
+            border-radius: 25px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 100%;
+            color: hsl(var(--accent-foreground));
+            background-color: hsl(var(--accent));
```

Comment:
It will be nice to add a transition here to animate the bar. The thing is that animating using `clip-path` looks weird. Is the usage of `clip-path` necessary? Can we avoid it?

Reply:
As it is it moves just a little every minute for real uses. Keep in mind currently mainnet pushes one batch ever 6 hours, any animation there would be unnoticeable. The  `clip-path` is what makes the progress bar work at all, as it is two halves inside a div that have different style, rather than a "real" progress bar. I could try to do something like that with widths but I don't know if it will work.